security/tools/root_transition/root-transition.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
```

Comment:
add instructions how to run this script on the top?

Reply:
Done in another PR.